bindings/nodejs/src/lib.rs

```diff
@@ -35,17 +35,53 @@ impl Memory {
 
     #[napi]
     pub fn build(&self) -> Operator {
-        self.make_operator()
+        self.make_operator().unwrap()
+    }
+}
+
+#[napi]
+pub struct Fs {
+    _root: String,
+}
+
+#[napi]
+impl Fs {
+    #[napi(constructor)]
+    pub fn new() -> Self {
+        Self { _root: String::from("") }
+    }
+
+    #[napi(setter)]
+    pub fn root(&mut self, root: String) {
```

Comment:
Export every function of services could be a boring job. And I want to keep the binding's simple and easy to maintain.

Maybe it's better to accept a scheme and a map instead? For example:

```rust
pub fn new(scheme: Scheme, cfg: HashMap<String, String>) -> Operator {
    todo!()
}
```

Reply:
SGTM. But we still need to check every prop whether users provide it or not.

Reply:
s3 service has 10+ configs :rofl:

Reply:
> we still need to check every prop whether users provide it or not.

No, we don't need to. We are just a binding, `ServiceBuilder` should handle this.

Reply:
After this change, we don't need to export any services any more~